backend/package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@azure/identity": "^3.1.2",
     "@azure/keyvault-secrets": "^4.6.0",
+    "axios": "^1.7.2",
```

Comment:
Why is axios needed also on backdend? Isn't Undici enough? Ideally only one library should be used, if possible.